variables.tf

```diff
@@ -135,3 +135,9 @@ variable "volume_type" {
   type = string
   default = ""
 }
+
+variable "compute_tags" {
+  description = "Set of tags to be applied to all compute instances"
```

Comment:
Could we make the variable name "instance_tags" (and update the description to match)?

All the instances are VMs but not all of them are really _compute_ VMs. Terraform calls them compute instances but in openstack some of them will be controllers, some will be compute nodes, some will be storage etc. As you can see it might get confusing.